agenda/presentation/src/main/java/com/gkp/agenda/presentation/edit/EditAgendaItemViewModel.kt

```diff
@@ -47,6 +52,7 @@ class EditAgendaItemViewModel(
             when (agendaItemType) {
                 AgendaItemType.REMINDER -> agendaRepository.fetchAgendaItemReminder(it)
                 AgendaItemType.TASK -> agendaRepository.fetchAgendaItemTask(it)
+                AgendaItemType.EVENT -> flowOf()
```

Comment:
FYI, this deserves a `// TODO: ` in a real-life project

Even better, you can use the following, which will crash the app with the exception message to ensure the app does not get shipped without implementing this `when` case. 

```
TODO(“Events not implemented in AgendaRepository”)
```